BUILD.bazel

```diff
@@ -370,6 +405,14 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "core_worker_test",
```

Comment:
cc_test?

Reply:
This is currently started by a script and cannot run by itself, it will be automatically run if it's a `cc_test`.  FYI this is not adding a new test, it just moves the files to a different folder.